include/object/structures_32.bf

```diff
@@ -273,31 +273,36 @@ tagged_union lookup_fault lufType {
 
 -- Fault: size = 8 bytes
 block NullFault {
+    padding 32
     padding 60
     field seL4_FaultType 4
 }
 
 block CapFault {
+    padding 32
     field address 32
     field inReceivePhase 1
     padding 27
     field seL4_FaultType 4
 }
 
 block UnknownSyscall {
+    padding 32
```

Comment:
There's a comment on the `struct tcb` definition that claims seL4_FaultType is 2 words which may need updating.